.gitignore

```diff
@@ -109,3 +109,13 @@ wheelhouse/
 # by default, ignore in-toto metadata from the command line
 .links/
 *.link
+n2ws/tests/__init__.py
+*.yaml
+n2ws/CHANGELOG.md
+*.py
+*.example
+n2ws/MANIFEST.in
+n2ws/requirements-dev.txt
+n2ws/requirements.in
+n2ws/tox.ini
+n2ws/metadata.csv
```

Comment:
Please, do not add these to the global git ignore. Specially the generic *.py and *.yaml